README.md

```diff
@@ -11,9 +11,7 @@ This application defines global templates for [GOV.UK](https://www.gov.uk) pages
 
 This is a Ruby on Rails app, and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html).
 
-You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) or the local `startup.sh` script to run the app. Read the [guidance on local frontend development](https://docs.publishing.service.gov.uk/manual/local-frontend-development.html) to find out more about each approach, before you get started.
-
-If you are using GOV.UK Docker, remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
+You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-docker) to run the app. Remember to combine it with the commands that follow. See the [GOV.UK Docker usage instructions](https://github.com/alphagov/govuk-docker#usage) for examples.
```

Comment:
Ideally we should make this the same as [all the other non-frontend repos](https://github.com/alphagov/email-alert-api#technical-documentation).